pythran/spec.py

```diff
@@ -300,6 +308,25 @@ def spec_expander(args):
     return all_specs
 
 
+def multiexports_to_exports(exports):
+    def decompose(value, arr, new_array=[]):
```

Comment:
you have a side effect on new_array *and* it's a default parameter. That's dangerous because of that http://docs.python-guide.org/en/latest/writing/gotchas/ even if that looks ok in your case. Please don't.